src/covid_shared/cli_tools.py

```diff
@@ -29,6 +29,20 @@
 }
 
 
+def _remove_secrets(dictionary: Dict, secrets: List[str]) -> Dict:
+    """
+    Remove secrets from a nested Dict
+    """
+    if secrets:
+        for key, value in dictionary.items():
+            if isinstance(value, dict):
+                dictionary[key] = _remove_secrets(value, secrets)
+            else:
+                if value in secrets:
```

Comment:
Is the intention to censor here based on the value, or key? I had thought that `secrets` would be a list like ['password',...] which would imply we want `key` instead of `value` here, no?

Reply:
The intention is to censor the values with secrets being a list.

For example, If the value is any one of the secrets, replace it with the string "***censored***"

Reply:
Ok so the usage is something like `_remove_secrets(some_dict, secrets=[os.environ.get('password')...])`?

Reply:
Yes. Any value in the nested Dict that matches any one of the secrets would be replaced with censored.

I'm using it like:
```
            secrets = list()
            secrets.append(ctx.obj['blob'].account_key)
            secrets.append(ctx.obj['batch'].account_key)
            secrets.append(ctx.obj['registry'].password)
            run_metadata.dump(metadata_path, secrets)
```